R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
```

Comment:
Let's keep the name similar to the `lubridate` method.

```suggestion
    "datetime",
```

Reply:
I did not see the openapi definition of `date-time` in the issue. 

Resolving to remove the suggestion.